tests/performance/apdex-score/test/page-objects/login.page.js

```diff
@@ -3,23 +3,23 @@ const Page = require('./page');
 
 class LoginPage extends Page {
   
-  get inputUsername () {
+  get inputUsername() {
     return $('//android.view.View[1]/android.widget.EditText');
   }
 
-  get inputPassword () {
+  get inputPassword() {
     return $('//android.view.View[3]/android.widget.EditText');
   }
```

Comment:
I think that creating a selector referring the element number might cause problems in the future. Please check if you can use text, for example, to access the element in a better way.

Also, maybe we can create a method in `page.js` that returns `inputs`, for example, and you only need to send as a parameter the text of the input that you need. Something similar to what we did on the [common-enketo.wdio.page.js ](https://github.com/medic/cht-core/blob/master/tests/page-objects/default/enketo/common-enketo.wdio.page.js) file

Reply:
Done thanks